plaaces-to-go/src/pages/menu.js

```diff
@@ -0,0 +1,59 @@
+import StyleMenu from '../styles/Menu.module.css';
+import React, {useState} from 'react';
+
+export default function Navbar() {
+    const [isOpen, setIsOpen] = useState(true)
+    return (
+        <nav className={StyleMenu.navbar}>
+            <div className={StyleMenu.navLogo}>
+                <p className={StyleMenu.txtItem}>
```

Comment:
You generally don't want to have to add classes to each single paragraph, it's generally a work for globals to define that kind of default styles

Reply:
Ok I got it :D